src/os/ui/screenoverlay.js

```diff
@@ -64,7 +64,8 @@ os.ui.launchScreenOverlay = function(options) {
     'show-hide': !!options.showHide,
     'show-close': !!options.showClose,
     'no-scroll': true,
-    'overlay': true
+    'overlay': true,
+    'border': '0'
```

Comment:
`'0'` is a poor value here. Another developer might assume that you mean to specify the border width (in which case an actual number would be better). However, it appears that you mean for this to be a boolean and this should be `false`.